README.md

```diff
@@ -12,6 +12,8 @@ Als je een stapje verder wilt gaan dan in de browser iedere file te wijzigen, ka
 
 Eenmaal je vertrouwd bent met hoe github op jouw machine werkt moet je Jekyll installeren, de statische site generator. Deze kan je hier vinden: https://jekyllrb.com/
 
+When building the website, use `bundle exec jekyll serve --future true` otherwise future events won't be rendered.
```

Comment:
There’s a difference between the date of an event and a date of when the event was published. You have to use a different property for the date/time of the actual event